static/js/taxpayer_creation.js

```diff
@@ -4,12 +4,64 @@ var left, opacity, scale;
 var animating; 
 
 
-$(".next").click(function(){
+// Checks if first part of form is complete
+function check_first_fieldset_complete(){
+	var token = true
+	$(".mandatory>input:text, .mandatory>input[type='number'], .mandatory>select, .mandatory>input[type='email']").each(function(index){
+		if ($(this).val() == "") {
+			token = false;
+		}
+		else if($(this).get(0).checkValidity() == false){
+			token = false;
+		}
+	});
+	return token
+}
+
+// Click on First "next" button
+$("#first-next").click(function(){
+	var esto = $(this);
+	if (check_first_fieldset_complete()){
+		next_page(esto);
+	}
+	else{
+		alert("Porfavor termine de completar el formulario antes de continuar");
+	}
+});
+
+// Checks if argentinian BANK form is filled
+function check_arg_bank_complete(){
```

Comment:
Same as one of the above comments this function follows the same logic as `check_first_fieldset_complete ` only changing the below selector maybe we can think in a function that receives a selector and execute this logic.